Sources/CollectionViewDriver.swift

```diff
@@ -197,7 +195,7 @@ public class CollectionViewDriver: NSObject, UICollectionViewDataSource, UIColle
             viewModel.applyViewModelToView(view)
             view.accessibilityIdentifier = viewModel.viewInfo?.accessibilityFormat.accessibilityIdentifierForSection(section)
         } else {
-            view = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: CollectionViewDriver._hiddenSupplementaryViewIdentifier, for: indexPath)
+            view = UICollectionReusableView()
```

Comment:
temp hack for now. we'll remove like we did with cells. but other things need to be fixed first